frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentControlButtons.tsx

```diff
@@ -81,12 +79,7 @@ export const PublicEnrollmentControlButtons = ({
             variant: Variant.Contained,
             action: () => {
               dispatch(confirmAction);
-              dispatch(resetPublicEnrollment());
```

Comment:
Tälle oli muistaakseni ainakin aiemmin ihan tarve olla täällä, mutta en nyt tiedä onko tuo tarve poistunut. Tätä kutsutaan myös muissa vast. tilanteissa, ainakin kun ilmoittautumisen peruuttaa varausajan loppumismodaalista tai Valmis-sivulta käsin.

Reply:
Joo, itsekin muistelin näin mutta ehkä jossain kohtaa tuo muuttunut tarpeettomaksi.

Tuolla `PublicEnrollmentGrid` tehdään tuo reset:
```typescript
    if (cancelStatus === APIResponseStatus.Success) {
      navigate(AppRoutes.PublicHomePage);
      dispatch(resetPublicEnrollment());
    }
```

Testasin myös tuon loppumismodaalin ja näytti toimivan. Mutta testaa toki huolella kun en oo varma löysinkö kaikki skenaariot.

Reply:
Tuota resettiä ei siis saada tehdä liian aikaisin tai muuten tuo navigation protection hyppää silmille.

Reply:
Jes. Tuon kutsun sais varmaan sitten myös PublicEnrollmentTimerin alta tiputettua pois? Done-stepissä ehkä tarpeellinen kun ei rendata enää ko. PublicEnrollmentGridiä?

Reply:
Tsekkaan noi vielä

Reply:
Jos commitoit tuon muutoksen vielä niin vois mun puolesta sen jälkeen mergetä